### PR TITLE
Alignment fix for abandoned zones

### LINKED ISSUE #1339: Fix pagination number in abandoned zones view
**Describe the bug**
Please provide as much detail as you can. Here are some important details:

1. The pagination number is not properly aligned in abandoned zones view as seen in the screenshot attached. It should be inline with the pagination buttons as seen in another screenshot attached.
Screenshot 2024-02-06 at 6 23 42 PM
Screenshot 2024-02-06 at 6 24 18 PM

### PR DESCRIPTION
**Pull request by Jay07GIT**
Fixes #1339.

Changes in this pull request:
- Alignment fix for abandoned zones.

### CODE REVIEW

**Review by nspadaccino — APPROVED** [review]
LGTM

**Review by nspadaccino — CHANGES_REQUESTED** [review]
Screenshot 2024-02-27 at 3 08 47 PM
Still looking mis-aligned for me

**Review by Aravindh-Raju — CHANGES_REQUESTED** [review]
As Nick mentioned, still mis-aligned for me as well.

File: modules/portal/app/views/zones/zones.scala.html [hunk 1]
```diff
@@ -308,170 +308,168 @@ <h3><span class="fa fa-table"></span> Zones</h3>
...
-                                                                <a ng-if="nextPageEnabled('myDeletedZones')" ng-click="nextPageMyDeletedZones()">Next</a>
-                                                            </li>
-                                                        </ul>
-                                                    </div>
-                                                    <!-- END PAGINATION -->
-
-                                                </div>
+                                        <span class=" container-fluid" >
+                                           <!-- MY DELETED ZONE PAGINATION -->
+                                            <div class="dataTables_paginate vinyldns_zones_paginate" ng-show="tab == 'myDeletedZones'">
```
Comment (Aravindh-Raju) [inline modules/portal/app/views/zones/zones.scala.html:313]: The `ng-show` condition displays the pagination button whenever we are on the tab. But generally, the button needs to show up only when we have reached the pagination limit set.

I looked into the code. We can revert all these changes and just change the following in all occurrences of `myDeletedZones` and `allDeletedZones` pagination html:
```
<div class="dataTables_paginate vinyldns_paginate">
```

We just have to replace `vinyldns_zones_paginate` with `vinyldns_paginate` for the page number to be aligned in `myDeletedZones` and `allDeletedZones`. There is no css class named `vinyldns_zones_paginate` which causes the misalignment.  I think we can revert all other changes made.

**Review by Aravindh-Raju — CHANGES_REQUESTED** [review]

File: modules/portal/app/views/zones/zones.scala.html
[the remark this answers is not in the document]
Comment (Jay07GIT) [reply modules/portal/app/views/zones/zones.scala.html:1]: Fixed the issue with tags instead of css changes. Thanks for suggestions.

**Review by arpit4ever — APPROVED** [review]
LGTM

**Review by Aravindh-Raju — APPROVED** [review]
LGTM!

**Review by nspadaccino — APPROVED** [review]
looks good

**Pull request merged**